src/components/Shared/Filters/FilterPanel.tsx

```diff
@@ -559,8 +559,7 @@ export const FilterPanel: React.FC<FilterPanelProps & BoxProps> = ({
   };
 
   const handleClearAllClick = () => {
```

Comment:
Could we remove this function entirely and just pass `clearFilters` directly to the link component?